pkg/hoist/successful_scripts_test_hoist_launchable/installs/testLaunchable_abc123/bin/disable

```diff
@@ -1,2 +1,4 @@
 #!/bin/bash
+echo "sleeping for 2 second to test termination grace period"
+sleep 0.005
```

Comment:
The echo and sleep statements don't line up. I'm good with the test being so fast that the `sleep` can be super short, but can we make the `echo` match the reality?